subprojects/gradle/runner/service/src/main/kotlin/com/avito/runner/service/worker/device/adb/AdbDevice.kt

```diff
@@ -387,12 +408,6 @@ data class AdbDevice(
         command = command
     )
         .ofType(ProcessNotification.Exit::class.java)
-        .doOnError { throwable ->
-            throw RuntimeException(
-                "Failed to execute command: $command on device $id",
-                throwable
-            )
-        }
```

Comment:
Useless wrapper for an error.

```
throwable = {CompositeException@34207} "rx.exceptions.CompositeException: 2 exceptions occurred. "
 exceptions = {Collections$UnmodifiableRandomAccessList@36729}  size = 2

  0 = {IllegalStateException@36833} "java.lang.IllegalStateException: Process [/home/user/Android/Sdk/platform-tools/adb, -s, emulator-5554, pull, /sdcard/Android/data/com.avito.android.ui/files/runner/com.avito.android.ui.test.DisplayedWithTextTest#canAssertText_onFoundItem, /tmp/tmp4294522707592643471.tmp] exited with non-zero code 1" 
  ^--- The original error

  1 = {RuntimeException@36834} "java.lang.RuntimeException: Failed to execute command: [pull, /sdcard/Android/data/com.avito.android.ui/files/runner/com.avito.android.ui.test.DisplayedWithTextTest#canAssertText_onFoundItem, /tmp/tmp4294522707592643471.tmp] on device emulator-5554"
 message = "2 exceptions occurred. "
  ^--- The wrapper
```


Motivation:
- Original error from ADB contains all the information
- Logger in AdbDevice knows about serial and adds it to a message. No need for duplication.